lib/fruitbot/commands.ex

```diff
@@ -1,36 +1,52 @@
 defmodule Fruitbot.Commands do
   def handle_message(payload) do
-    IO.inspect payload
-    [ command | tail ] = String.split(payload)
-    args = Enum.join(tail)
+    IO.inspect(payload)
+    [command | tail] = String.split(payload)
+    query = Enum.join(tail)
+
     case command do
       "!vr" ->
-        msg = "Join the party in VR here! https://hubs.mozilla.com/MsvfAkH/terrific-satisfied-area"
-        { :ok, msg }
+        msg =
+          "Join the party in VR here! https://hubs.mozilla.com/MsvfAkH/terrific-satisfied-area"
+
+        {:ok, msg}
+
       "!donate" ->
-        msg = "Enjoying the stream? The best way to support is with a monthly donation on ampled. https://www.ampled.com/artist/datafruits You can also make a one time donation to our paypal. https://paypal.me/datafruitsfm"
-        { :ok, msg }
+        msg =
+          "Enjoying the stream? The best way to support is with a monthly donation on ampled. https://www.ampled.com/artist/datafruits You can also make a one time donation to our paypal. https://paypal.me/datafruitsfm"
+
+        {:ok, msg}
+
+      # TODO would it be fun to have !advice randomly draw from the datafruits marquee or something? I mean I like the cheese cow advice but everytime
```

Comment:
Using the marquee texts is a cool idea.

The other idea we had was pull some other coach mcguirk quotes from a database.